ansible/collections/bento/common/roles/opensearch_dataloading/files/opensearch-restore.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import boto3
+import requests
+from requests_aws4auth import AWS4Auth
+import time
+
+parser = argparse.ArgumentParser(description='Opensearch Backup Script')
+parser.add_argument("--oshost", type=str, help="opensearch host with trailing /")
+parser.add_argument("--repo", type=str, help="opensearch snapshot repository")
+parser.add_argument("--s3bucket", type=str, help="s3 bucket")
+parser.add_argument("--snapshot", type=str, help="opensearch snapshot value")
+parser.add_argument("--indices", type=str, help="indices")
+parser.add_argument("--rolearn", type=str, help="role arn - typically power user role")
+parser.add_argument("--basepath", type=str, help="basepath")
+args = parser.parse_args()
+oshost = args.oshost
+repo = args.repo
+s3bucket= args.s3bucket
+snapshot= args.snapshot 
+indices = args.indices
+rolearn = args.rolearn 
+basepath = args.basepath
+# test
+host = oshost 
+region = 'us-east-1'
```

Comment:
can the region be passed in - we always use the same region right now, but that may not always be the case

Reply:
region = boto3.Session().region_name